docs/developer-guide/new-role.rst

```diff
@@ -0,0 +1,50 @@
+.. Copyright (C) 2021 Robin Schneider <ypid@riseup.net>
+.. Copyright (C) 2021 DebOps <https://debops.org/>
+.. SPDX-License-Identifier: GPL-3.0-or-later
+
+Creating a new DebOps role
+==========================
+
+Historically, a new role was created by copying an existing role and doing an
+initial search and replace. Which has the following problems:
+
+- Which role is up to the latest standards and thus suitable for creating the next role?
+- Requires manual work to setup a new role skeleton before work can be started (search and replace).
+- For new contributers, this is even more difficult.
+- The role standards are going to evolve thus the roles need to be updated.
+
+DebOps uses `Copier`__ to help address those issues.
+
+  .. __: https://copier.readthedocs.io/en/stable/
+
+Setup
+-----
+
+- Install copier.
+- Clone https://github.com/ypid/debops-template.git next to the debops monorepo (or symlink it next to it).
+
+Create a new rule
+-----------------
+
+In :file:`ansible/roles`, run:
+
+.. code-block:: shell
+
+   copier -d "author_fullname=$(git config --get user.name)" -d "author_email=$(git config --get user.email)" copy ../../../debops-template/ new_role
```

Comment:
I assume new_role is the short name of the new role ? this probably needs to be stated explicitely (esp. if there are naming rules...)

Reply:
I updated the docs to indicate that this will create a role called `new_role` which should obviously be replaced.